packages/components/toggle/src/toggle.test.ts

```diff
@@ -37,4 +38,26 @@ describe('OkuToggle', () => {
     await wrapper.trigger('click')
     expect(wrapper.attributes('aria-pressed')).toBe('false')
   })
+
+  it('v-model', async () => {
+    const component = {
+      template: `
+          <OkuToggle v-model="value" />
+        `,
+      components: {
+        OkuToggle,
+      },
+      setup() {
+        const value = ref(false)
+        return {
+          value,
+        }
+      },
+    }
```

Comment:
I think it's better.

https://test-utils.vuejs.org/guide/advanced/v-model.html

Reply:
Yes we can implement this, let's think about it in the next tests.